back/src/main/java/com/baba/back/content/service/ContentService.java

```diff
@@ -74,4 +81,31 @@ private void checkAuthorization(Relation relation) {
             throw new ContentAuthorizationException(relation.getId() + " 관계는 가족 관계가 아닙니다.");
         }
     }
+
+    public LikeContentResponse likeContent(String memberId, String babyId, Long contentId) {
+        final Member member = findMember(memberId);
+        final Baby baby = findBaby(babyId);
+        findRelation(member, baby);
+        final Content content = findContent(contentId);
+        final Like like = findAndUpdateLike(member, content);
+
+        likeRepository.save(like);
+
+        return new LikeContentResponse(!like.isDeleted());
+    }
+
+    private Content findContent(Long contentId) {
+        return contentRepository.findById(contentId)
+                .orElseThrow(() -> new ContentNotFountException(contentId + " 는 존재하지 않는 컨텐츠입니다."));
+    }
+
+    private Like findAndUpdateLike(Member member, Content content) {
+        final Optional<Like> like = likeRepository.findByMemberAndContent(member, content);
+        like.ifPresent(BaseEntity::updateDeleted);
```

Comment:
```suggestion
        like.ifPresent(Like::updateDeleted);
```
BaseEntity를 직접 호출하기보다는 LIke를 호출하는 것이 더 적절한 것 같습니다.

Reply:
수정할게요~!